helix-rest/src/test/java/org/apache/helix/rest/server/service/TestInstanceService.java

```diff
@@ -140,6 +151,70 @@ public void testGetInstanceStoppableCheckConnectionRefused() throws IOException
     Assert.assertEquals(actual.getFailedChecks().get(0), expectedFailedCheck);
   }
 
+  @Test
+  public void testCustomPartitionCheckWithSkipZKRead() throws IOException {
+    // Let ZK result is health, but http request is unhealthy.
+    // We expect the check fail if we skipZKRead.
+
+    String testPartition = "PARTITION_0";
+    String slibilingInstance = "instance0.linkedin.com_1236";
+    String jsonContent = "{\n" + "   \"param1\": \"value1\",\n" + "\"param2\": \"value2\"\n" + "}";
```

Comment:
Why does this jsonContent need multiple string concatenations?

Reply:
Was for viewing purpose for each entry set a line.